modules/ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id='ingress-setting-a-custom-default-certificate-{context}']
+= Setting a Custom Default Certificate
+
+You can configure an ingress controller to use a custom certificate by creating a `Secret` resource and editing the `IngressController` custom resource (CR).
+
+.Prerequisites
+
+* You must have a certificate/key pair in PEM-encoded files, where the certificate is signed by a trusted certificate authority and valid for the ingress domain.
+
+* You must have an `IngressController` CR.  You may use the default one:
++
+----
+$ oc -n openshift-ingress-operator get IngressControllers
```

Comment:
1. I've been using long-form args (e.g. `--namespace`) — what's the convention?
2. Camel-cased resource names on the CLI looks really out of place — what's the convention?
3. What about getting the default one directly rather than showing a list?